dev/client.scss

```diff
@@ -14,13 +14,20 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+*,
+::after,
+::before {
+    box-sizing: border-box;
+}
+
 html,
 body {
     padding: 0;
     margin: 0;
 }
 
 #app {
+    background: #ffffff;
```

Comment:
Shorthand to `#fff`?

Reply:
And prefer specific property of `background-color`?

Reply:
stylelint here requires all hex codes to be in 6 char form

Reply:
Not sure why there'd be preference for `background-color` over the generic `background`?

Reply:
I always prefer to be more specific to guard against the chance of it overriding another declaration incidentally and to provide consistent legibility. It's a preference. Happy to default to this repo's standard.

Reply:
Ah, interesting. I've always leaned toward using shorthand properties when they're available for conciseness and to avoid property collisions (the same reason you have for not doing it :lul: -- if you have both `background-color` and `background-image` set, I think it's not always apparent what's happening when compared to defining both in a single `background`).

Reply:
As I've personally always leaned toward alphabetizing properties, I find either way legible; depends on the way you read I suppose :). I'm happy to keep it this way and not impose my personal preference.